assets/js/modules/analytics/components/dashboard/DashboardAllTrafficWidgetV2/TotalUserCount.js

```diff
@@ -20,13 +20,112 @@
  * External dependencies
  */
 import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import { STORE_NAME as CORE_SITE } from '../../../../../googlesitekit/datastore/site/constants';
+import { STORE_NAME as CORE_USER } from '../../../../../googlesitekit/datastore/user/constants';
+import { STORE_NAME, DATE_RANGE_OFFSET } from '../../../datastore/constants';
+import { numberFormat, readableLargeNumber } from '../../../../../util';
+import { getAvailableDateRanges } from '../../../../../util/date-range';
+import { isZeroReport } from '../../../util';
+import ChangeArrow from '../../../../../components/ChangeArrow';
+const { useSelect } = Data;
 
-// eslint-disable-next-line no-unused-vars
 export default function TotalUserCount( { dimensionName, dimensionValue } ) {
-	return null;
+	const url = useSelect( ( select ) => select( CORE_SITE ).getCurrentEntityURL() );
+	const dateRange = useSelect( ( select ) => select( CORE_USER ).getDateRange() );
+	const dateRangeDates = useSelect( ( select ) => select( CORE_USER ).getDateRangeDates( {
+		compare: true,
+		offsetDays: DATE_RANGE_OFFSET,
+	} ) );
+
+	const args = {
+		...dateRangeDates,
+		metrics: [ { expression: 'ga:users' } ],
+		orderby: {
+			fieldName: 'ga:users',
+			sortOrder: 'DESCENDING',
+		},
+		limit: 1,
+	};
+
+	if ( url ) {
+		args.url = url;
+	}
+
+	if ( dimensionName && dimensionValue ) {
+		args.dimensions = [ dimensionName ];
+		args.dimensionFilters = {
+			[ dimensionName ]: dimensionValue,
+		};
+	}
+
+	const loaded = useSelect( ( select ) => select( STORE_NAME ).hasFinishedResolution( 'getReport', [ args ] ) );
+	const error = useSelect( ( select ) => select( STORE_NAME ).getErrorForSelector( 'getReport', [ args ] ) );
+	const report = useSelect( ( select ) => select( STORE_NAME ).getReport( args ) );
+
+	if ( ! loaded || error || isZeroReport( report ) ) {
+		return null;
+	}
+
+	const { totals } = report?.[ 0 ]?.data || {};
+	const [ current, previous ] = totals || [];
+	const change = previous?.values?.[ 0 ] > 0
+		? ( current?.values?.[ 0 ] / previous?.values?.[ 0 ] ) - 1
+		: null;
```

Comment:
This should be using the `calculateChange` function which returns a value between 0 and 1.

Reply:
Updated.